libs/partners/ollama/langchain_ollama/chat_models.py

```diff
@@ -476,15 +476,16 @@ async def _acreate_chat_stream(
 
         params["options"]["stop"] = stop
         if "tools" in kwargs:
-            yield await self._async_client.chat(
+            async for part in await self._async_client.chat(
```

Comment:
Do we need to also fix the async streaming case? We unfortunately don't cover async streaming with tools in our standard tests.

Reply:
I added a standard test for async tool calling this morning in https://github.com/langchain-ai/langchain/pull/28133. You can run it against Ollama via
```
python -m pytest tests/integration_tests/test_chat_models.py::TestChatOllama::test_tool_calling_async
```
It appeared broken here due to changing `stream` to True in the async case for tool calling. I pushed an update to fix the test. Confirmed no new failures across standard tests.